stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md

```diff
@@ -0,0 +1,191 @@
+---
```

Comment:
rm `&` from filename

Reply:
I remove the '&' and renamed the file file name